From fe2f94e1235ddc2d24d05b58fef39e7b352e3cc1 Mon Sep 17 00:00:00 2001
From: Caster <caster@gentoo.org>
Date: Wed, 24 Jun 2009 14:38:23 +0200
Subject: [PATCH 3/3] disable core updater

Provided by Hanno Meyer-Thurow <h.mth@web.de> in bug #218959.
---
 .../pluginsimpl/local/PluginInitializer.java       |   12 ------------
 1 files changed, 0 insertions(+), 12 deletions(-)

diff --git a/org/gudy/azureus2/pluginsimpl/local/PluginInitializer.java b/org/gudy/azureus2/pluginsimpl/local/PluginInitializer.java
index efc11e9..b450ea7 100644
--- a/org/gudy/azureus2/pluginsimpl/local/PluginInitializer.java
+++ b/org/gudy/azureus2/pluginsimpl/local/PluginInitializer.java
@@ -125,18 +125,6 @@ PluginInitializer
 					"Magnet URI Handler",
 					"true",
 					"false"},
-			{	 PluginManagerDefaults.PID_CORE_UPDATE_CHECKER, 
-   					"org.gudy.azureus2.update.CoreUpdateChecker", 
-   					"azbpcoreupdater", 
-   					"CoreUpdater",
-					"true",
-					"true"},
-			{	 PluginManagerDefaults.PID_CORE_PATCH_CHECKER, 
-   					"org.gudy.azureus2.update.CorePatchChecker", 
-   					"azbpcorepatcher", 
-   					"CorePatcher",
-					"true",
-					"true"},
 	   		{	 PluginManagerDefaults.PID_PLATFORM_CHECKER, 
    					"org.gudy.azureus2.platform.PlatformManagerPluginDelegate", 
    					"azplatform2", 
-- 
1.6.3.1

